src/Nethermind/Nethermind.BeaconNode/Services/IClock.cs

```diff
@@ -4,6 +4,6 @@ namespace Nethermind.BeaconNode.Services
 {
     public interface IClock
```

Comment:
Note that this is something that may end up in a common library of services, across beacon node and honest validator, as both may want to share some common services like a clock interface. (I don't suggest doing it now, as we don't need it yet; just a comment)

Reply:
We have a Timestamper in Eth1 but prefer to keep them separate for now.